docker/experimental/terraform/modules/trigger/main.tf

```diff
@@ -0,0 +1,65 @@
+variable "release" {
+  type = string
+}
+
+variable "python_version" {
+  description = "Python version to use (e.g. 3.8)"
+  type = string
+}
+
+variable "platform" {
+  type = string
+}
+
+variable "docker_build_args" {
+  type = list(string)
+  default = [ "tpuvm=1" ]
+}
+
+locals {
+  trigger_name = format("pytorch-xla-%s-py%s-%s", var.release, replace(var.python_version, ".", ""), var.platform)
+}
+
+resource "google_cloudbuild_trigger" "build-trigger" {
+  location = "global"
+  name = local.trigger_name
+
+  source_to_build {
+    uri = "https://github.com/pytorch/xla"
+    repo_type = "GITHUB"
+    # TODO: make branch configurable
+    ref = "refs/heads/master"
+  }
+
+  git_file_source {
+    path = "docker/experimental/cloudbuild.yaml"
+    repo_type = "GITHUB"
+    # TODO: make branch configurable
+    revision = "refs/heads/master"
+    uri = "https://github.com/pytorch/xla"
+  }
+
+  substitutions = {
+    _PLATFORM = var.platform
+    _BUILD_ARGS = join(",", var.docker_build_args)
+    _PYTHON_VERSION = var.python_version
+  }
+}
+
+resource "google_cloud_scheduler_job" "trigger-schedule" {
+  name = format("%s-schedule", local.trigger_name)
+  region = "us-central1"
+
+  schedule = "0 0 * * *"
```

Comment:
Could you add a comment explaining what the schedule string means?

Reply:
Linked to Cloud Scheduler's documentation on cron schedule format.